### PR TITLE
Feature/poll

### PR DESCRIPTION
**Pull request by smikayel**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
     image: mysql:5.7
     restart: always
     environment:
-      MYSQL_DATABASE: 'db'
+      MYSQL_DATABASE: 'poll_app'
```
Comment (GagMirz) [inline docker-compose.yml:7]: No need to change this

File: users/views.py [hunk 2]
```diff
@@ -1,45 +1,135 @@
...
+                creator = creator
+            )
+            return  JsonResponse(model_to_dict(createdPOLL),status=200)
         except Exception as e:
             print(e)
+            return JsonResponse({"data": "", "error": "Something went wrong"}, status=500)
+
+    def get(self, request):
+        lst = Poll.objects.all().values()
+        return JsonResponse({'data': list(lst)}, status = 200)
```
Comment (GagMirz) [inline users/views.py:109]: add paging to polls, to not get few hundreds polls at a time, paging and page size

File: users/models.py [hunk 3]
```diff
@@ -9,4 +11,19 @@ class User(models.Model):
...
\ No newline at end of file
+        return self.username
+
+
+class Poll(models.Model):
+    uuid = models.CharField(primary_key=True, max_length=255)
+    title = models.CharField(max_length=255)
+    firstOption = models.CharField(max_length=255)
+    secondOption = models.CharField(max_length=255)
+    voteCount = models.IntegerField(default=0)
```
Comment (GagMirz) [inline users/models.py:22]: need to separate vote counts, for first option and second option, update endpoints as well

Comment (smikayel) [reply users/models.py]: sorry but about endpoints what you mean, (urls or shomthing else need to update?)

**Pull request merged**